src/handlers/store-raw-invoice-textract-data/store-expense-documents.ts

```diff
@@ -40,11 +41,29 @@ export async function storeExpenseDocuments(
     return await handleTextractFailure(sourceBucket, sourceFilePath);
   }
 
+  const destinationFileName: string = (() => {
+    // I've done this rather than update the lib
+    // I assume es7 was mandated by someone at some point
+    const replaceAll = (
+      string: string,
+      searchValue: string | RegExp,
+      replaceValue: string
+    ): string => {
+      if (!string.match(searchValue)) return string;
+      return replaceAll(
+        string.replace(searchValue, replaceValue),
+        searchValue,
+        replaceValue
+      );
+    };
+    return replaceAll(sourceFileName.replace(/\.pdf$/g, ".json"), /\s/g, "_");
+  })();
+
   await handleTextractSuccess(
     sourceBucket,
     sourceFilePath,
     destinationBucket,
-    `${sourceFolder}/${jobId}.json`,
+    `${sourceFolder}/${destinationFileName}`,
```

Comment:
I made this change so that the names files created after Textract succeeds are the same as the pdf that they came from. I did this in part to facilitate testing as it means that we can have a given pdf file name and search for that in the output buckets.